pkg/chartmuseum/server/multitenant/index.go

```diff
@@ -40,9 +42,12 @@ func (server *MultiTenantServer) getIndexFile(log cm_logger.LoggingFn, repo stri
 	}
 
 	entry.RepoLock.Lock()
+
 	defer entry.RepoLock.Unlock()
-	// if cache is nil, and not on a timer, regenerate it
-	if len(entry.RepoIndex.Entries) == 0 && server.CacheInterval == 0 {
+	// if the keep-chart-always-update-to-date flag is set, we always update the index file
+	// and ignore the chart cache
+	if server.KeepChartAlwaysUpToDate /* the flag is set */ ||
```

Comment:
do we still need to check if the cache interval is set here?

Reply:
I think we  may need not , the new flag will override the behavior over the cacheInterval , and if we unset this flag the `cacheInterval` will still work , I think it is more easy-configuration , any thoughts ?